### DISCUSSION TITLE
How to navigate between tabs?

### DISCUSSION THREAD

**Discussion by ZhiyuanChen** [LunarVim/LunarVim#2969] [Q&A] [answered]
`SHIFT + h (H)` and `SHIFT + l (L)` seems not working

**Comment by okflea** [comment]
you need to uncomment a few lines that are already present in config.lua

**Comment by Lun4m** [comment, marked as answer]
Unfortunately those keymaps were recently removed in PR https://github.com/LunarVim/LunarVim/pull/2874.
You need to add
```lua
lvim.keys.normal_mode["<S-l>"] = ":BufferLineCycleNext<CR>"
lvim.keys.normal_mode["<S-h>"] = ":BufferLineCyclePrev<CR>"
```
in your `config.lua`.

**Reply by ZhiyuanChen** [reply]
Thank you~
